app/helpers/pickup_libraries_helper.rb

```diff
@@ -3,15 +3,17 @@
 ##
 # Helpers for formatting and displaying pickup libraries
 module PickupLibrariesHelper
-  def select_for_pickup_libraries(form)
-    pickup_libraries = form.object.pickup_libraries
-    return unless pickup_libraries.present?
+  include Folio::TypesUtils
+  # Pickup libraries for Symphony
+  def select_for_pickup_destinations(form)
```

Comment:
**suggestion** Can this be made into a viewComponent? ViewComponents use proper encapsulation, so they are easier to test and reason about than helpers.

Reply:
I may leave this to a future pull request.